src/core/utils/payment/stripe.ts

```diff
@@ -87,13 +97,20 @@ export async function createSubscription(
         billing_cycle_anchor: Math.floor(+renewalDate / 1000),
         proration_behavior: "none"
       })
-  });
+  };
+
+  if (defaultTaxRates) {
+    params.default_tax_rates = defaultTaxRates;
+  }
+
+  return await stripe.subscriptions.create(params);
 }
 
 export async function updateSubscription(
   subscriptionId: string,
   paymentForm: PaymentForm,
-  paymentMethod: PaymentMethod
+  paymentMethod: PaymentMethod,
+  defaultTaxRates?: string[]
```

Comment:
I don't think we should assume updates will happen this way, adding tax as people update their subscription seems like strange behaviour to me. Can you remove this until we've decided what happens to old subscriptions?

Reply:
okay, okay i'll leave the parameter in but remove that the default tax will be used and add a corresponding comment

Reply:
Is removed now